### PR TITLE
Changed package name and made client return errors correctly

### PR DESCRIPTION
**Pull request by ronanbarrett**
* Using new package name to avoid conflict with "mock"
* Using new repo name so we can open source soon
* Fixed error handling and improved response to help user fix issues

Note: Sequence is not implemented yet but will be soon.

### CODE REVIEW

**Comment by sfro** [comment]
I am so excited about open sourcing! :D

File: pkg/mockclient/client.go [hunk 1]
```diff
@@ -91,11 +93,18 @@ func (c *Client) callMock(mockAPI, mockReqBody string) {
...
+	if mockRes.StatusCode >= 200 && mockRes.StatusCode <= 299 {
+		return
+	}
+
+	// something went wrong so return the error message (Note: MockServer /verify returns 406 on failure)
+	b, err := ioutil.ReadAll(mockRes.Body)
+	if err != nil {
+		assert.Fail(c.T, fmt.Sprintf("MockServer call failed with status: %s", mockRes.Status))
 	}
+	assert.Fail(c.T, fmt.Sprintf("MockServer call to /%s failed with status: %s. Error message from MockServer is: %s", mockAPI, mockRes.Status, string(b)))
```
Comment (sfro) [inline pkg/mockclient/client.go]: Does this error message look nice with last string(b)? Do you get like a nice json that is easy to read?

Comment (ronanbarrett) [reply pkg/mockclient/client.go]: Hmm good point. It looks ok I'd say but i'll see if I can make it more beautiful.

Comment (ronanbarrett) [reply pkg/mockclient/client.go]: Screen Shot 2019-04-04 at 15 16 52

Actually it looks pretty ok.

Comment (sfro) [reply pkg/mockclient/client.go]: Nice! :D Was wondering since some json outputs look like a dumpster fire

Comment (ronanbarrett) [reply pkg/mockclient/client.go]: This is a backquote escaped, reasonably indented, small dumpster fire :)

**Pull request merged**